packages/cli/src/utils.ts

```diff
@@ -132,14 +132,32 @@ const checkImageExtension = (uri: string): boolean => {
   );
 };
 
+export const isDevnet = (rpcUrl: string): boolean => {
+  return rpcUrl.indexOf("devnet") != -1;
+};
+
+export const isTestnet = (rpcUrl: string): boolean => {
+  return rpcUrl.indexOf("testnet") != -1;
+};
+
+export const isMainnet = (rpcUrl: string): boolean => {
+  return rpcUrl.indexOf("mainnet") != -1;
+};
+
+export const checkSubmissionNetwork = (rpcUrl: string) => {
+  if (isDevnet(rpcUrl) || isTestnet(rpcUrl)) {
+    throw new Error("Thank you, but the Solana Mobile dApp publisher portal only supports mainnet submissions.");
```

Comment:
I'd recommend we put a bit more detail into this message, and call out explicitly what caused the failure. Something like:

> It looks like you are attempting to publish an app with a devnet or testnet RPC endpoint. Please ensure that your app NFTs (publisher, app, and release) are minted on mainnet beta, and re-run this command with a mainnet beta RPC endpoint.

WDYT?

Reply:
Do you think that is general to share the message across all of the portal-related commands - submit, update, etc?